os/mm/mm_heap/mm_mallinfo.c

```diff
@@ -73,9 +86,112 @@
  * Private Functions
  ****************************************************************************/
 
+static void dump_node(struct mm_allocnode_s *node, node_type_t type)
```

Comment:
How about making new file for new added? Because they are not mm_mallinfo.

Reply:
I think we can make mm_check_heap_corruption.c.

Reply:
Done.